File: src/main/java/org/interview/twitter/model/TwitterRequestDto.java
package org.interview.twitter.model;


import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

public class TwitterRequestDto implements Serializable {
    private String applicationName;
    private String pinID;
    private String consumerKey;
    private String consumerSecurityKey;

    @JsonProperty("pin")
    public String getPinID() {
        return pinID;
    }

    public void setPinID(String pinID) {
        this.pinID = pinID;
    }

    @JsonProperty("applicationName")
    public String getConsumerKey() {
        return consumerKey;
    }

    public void setConsumerKey(String consumerKey) {
        this.consumerKey = consumerKey;
    }

    @JsonProperty("applicationName")
    public String getConsumerSecurityKey() {
        return consumerSecurityKey;
    }

    public void setConsumerSecurityKey(String consumerSecurityKey) {
        this.consumerSecurityKey = consumerSecurityKey;
    }

    @JsonProperty("appName")
    public String getApplicationName() {
        return applicationName;
    }
    public void setApplicationName(String applicationName) {
        this.applicationName = applicationName;
    }

    @Override
    public String toString() {
        return "TwitterRequestDto{" +
                "applicationName='" + applicationName + '\'' +
                ", pinID='" + pinID + '\'' +
                ", consumerKey='" + consumerKey + '\'' +
                ", consumerSecurityKey='" + consumerSecurityKey + '\'' +
                '}';
    }
}

File: src/main/kotlin/org/interview/twitter/model/TwitterRequestDto.kt
package org.interview.twitter.model


import com.fasterxml.jackson.annotation.JsonProperty

import java.io.Serializable

/**
 * This is a Dto class
 *
 * @Author Mehdi
 */
data class TwitterRequestDto(@JsonProperty("applicationName") val applicationName: String?,
                             @JsonProperty("pinID") var pinID: String?,
                             @JsonProperty("consumerKey") var consumerKey: String?,
                             @JsonProperty("consumerSecurityKey") val consumerSecurityKey: String?,
                             @JsonProperty("keywordSearch") val keywordSearch: String?
                             ) :Serializable
